tools/cli/commands/build.js

```diff
@@ -768,13 +768,39 @@ async function buildProject( t ) {
 		if ( composerJson.repositories.length === 0 ) {
 			delete composerJson.repositories;
 		}
-		await writeFileAtomic(
-			`${ buildDir }/composer.json`,
-			JSON.stringify( composerJson, null, '\t' ) + '\n',
-			{ encoding: 'utf8' }
-		);
 	}
```

Comment:
In practice it probably doesn't matter, but we might put the new code inside this `if` block instead of after it (and keep the `writeFileAtomic` in the `if` block too). If somehow the project isn't using any monorepo packages, it doesn't seem like we'd need to bother rewriting versions either.

Reply:
c16c7a9aa069cc13f9c3401c57a27523c1fbb8da